lldb/unittests/Symbol/TestSwiftASTContext.cpp

```diff
@@ -199,3 +199,32 @@ TEST_F(TestSwiftASTContext, IsNonTriviallyManagedReferenceType) {
                                                                    nullptr));
 #endif
 }
+
+namespace {
```

Comment:
I wasn't sure where to put new testcase, and for the first attempt I've decided to place it near existing `SwiftASTContext` tests

Reply:
`lldb/unittest/Symbol/TestSwiftASTContext.cpp`

Reply:
It's already there :)